src/app/data-entries/dropdown/dropdown-tree-hierarchical-selection/dropdown-tree-hierarchical-selection.html

```diff
@@ -0,0 +1,20 @@
+<div class="button-wrapper">
+    <igx-chips-area>
```

Comment:
Wrap the chip area and button in a container which leaves empty space for the chip area when there's no chips in it. Also replace the raised button with an icon button with chevron down or something of that sort.